src/Catty UITests/ScriptCollectionView/BrickCellPickerViewerTest.swift

```diff
@@ -100,41 +100,16 @@ class BrickCellPickerViewerTests: XCTestCase {
             app.pickerWheels.element.adjust(toPickerWheelValue: testValues[0])
```

Comment:
And a last note. Maybe add a check e.g. AssertFalse here, in order to verify that testValue[0] is not currently selected?
Something like "XCTAssertFalse(waitForElementToAppear(app.otherElements[ACCESSABILITY_LABEL_VARIABLE + testValues[0]]).exists, "Brick: " + brick + " has wrong initial value)"